src/main/java/manga/Manga.java

```diff
@@ -1,12 +1,15 @@
 package manga;
 
 import author.Author;
+import com.google.gson.annotations.Expose;
 
 import java.util.Objects;
 
 public class Manga {
+    @Expose
```

Comment:
Comments would be helpful in describing what does `@Expose` achieves here.

Reply:
Will do at a later date.